### DISCUSSION TITLE
请问 PaddelOCR 可以识别文字之后去除图像上的文字么

### DISCUSSION THREAD

**Discussion by Charimanhua** [PaddlePaddle/PaddleOCR#14332] [Q&A]
如题，请问 PaddelOCR 可以识别文字之后去除图像上的文字么

**Comment by GreatV** [comment]
不行

**Comment by SWHL** [comment]
这个可以用文生图中inpainting的方案，大概思路是将文字框和对应prompt给到FLUX等inpainting模型，让它重新绘制即可。
给一个在线demo，可以参考：https://huggingface.co/spaces/ameerazam08/FLUX.1-dev-Inpainting-Model-Beta-GPU
